services/src/main/java/io/grpc/services/HealthCheckingLoadBalancerFactory.java

```diff
@@ -274,13 +279,16 @@ void setServiceName(@Nullable String newServiceName) {
       adjustHealthCheck();
     }
 
-    void updateRawState(ConnectivityStateInfo rawState) {
+    public void onSubchannelState(ConnectivityStateInfo rawState) {
```

Comment:
Isn't `HealthCheckState` implementing `SubchannelStateListener` any more?

Reply:
You are right. Made `HealthCheckState` implement `SubchannelStateListener` again.